src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -629,7 +629,7 @@ public void TestMinWorkingSet()
                 Assert.InRange((long)p.MinWorkingSet, 0, long.MaxValue);
             }
 
-            if (OperatingSystem.IsMacOS() || OperatingSystem.IsFreeBSD()) {
+            if (PlatformDetection.IsBsdLike || PlatformDetection.IsSunOS) {
```

Comment:
IsBsdLike includes more platforms than this code was tested against. I'd keep the existing part of the condition untouched.

Reply:
Are your sure?  I looked at the implementation of IsBsdLike() and it appeared to me that all the platforms that includes use the same get/set Min/Max WorkingSet code that only supports acting on the current process.

Reply:
OperatingSystem.IsMacOS() || OperatingSystem.IsFreeBSD()
does not include iOS, macCatalyst, NetBSD etc. which IsBsdLike is pulling.

Besides, anything which is not part of the port work is best left for other PRs. This is to make the review process easier for all parties.

Reply:
OperatingSystem.IsMacOS() || OperatingSystem.IsFreeBSD()
does not include iOS, macCatalyst, NetBSD etc. which IsBsdLike is pulling.

Besides, anything which is not part of the port work is best left for other PRs. This is to make the review process easier for all participants.

Reply:
Doing this stuff in a separate PR